docs/data/data-grid/demo/PopularFeaturesDemo.js

```diff
@@ -315,10 +334,14 @@ const columns = [
     type: 'singleSelect',
     valueOptions: ['Premium', 'Pro', 'Community'],
     renderCell: (params) => {
```

Comment:
[Side note] I feel like we miss the possibility to define `renderCell` for row cells only while keeping the default renderers for aggregation and grouping.
Maybe we should consider the `undefined` value returned by `renderCell` as if there was no `renderCell` at all?
This would look something like this:
```tsx
renderCell: (params) => {
  if (params.aggregation) {
    // Use default cell renderers for aggregation cells
    return undefined;
  }
  if (params.rowNode.type === 'group') {
    // Use default cell renderers for grouping cells
    return undefined;
  }
  return <CustomComponent {...params} />
}
```

Reply:
I like that solution, @cherniavskii.

> I feel like we miss the possibility to define renderCell for row cells only.

That would probably be the best DX (although it may be a harsh breaking change). I'd say that rendering the values on auto-generated rows like footer and row grouping is usually a separate case.